test/libsolidity/smtCheckerTests/functions/functions_library_internal.sol

```diff
@@ -0,0 +1,14 @@
+library L {
+	function value(function()internal a, uint256 b) internal {}
+}
+contract C {
+	using L for function()internal;
+
+	function f() public {
+		function()internal x;
+		x.value(42);
```

Comment:
What exactly was triggering the assertion before, was it `x.value`?

Reply:
yes

Reply:
I think I still don't fully understand the bug. The `MemberAccess` in the C++ code has function type, right? But it's not the function type of the variable `x`, it's the function type of `x.value`?

Reply:
Right. `MemberAccess` is about the `value` function. It's the function type of `value` or `x.value` function from library.

Reply:
@leonardoalt I was wrong about that. I updated PR description with hopefully true cause.